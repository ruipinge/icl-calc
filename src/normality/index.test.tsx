import TestRenderer from 'react-test-renderer';
import { Normality } from '.';

it('renders without crashing', () => {
  const tree = TestRenderer.create(<Normality />).toJSON();
  expect(tree).toMatchSnapshot();
});
