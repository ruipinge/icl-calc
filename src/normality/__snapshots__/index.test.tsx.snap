// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `"Normality Graphs are coming soon..."`;
